source/scale/performance-monitoring.rst

```diff
@@ -8,15 +8,39 @@ Performance monitoring
 
  <p class="mm-label-note">Also available in legacy Mattermost Enterprise Edition E20</p>
 
-Performance monitoring support enables a Mattermost server to track system health for large Enterprise deployments through integrations with `Prometheus <https://prometheus.io/>`__ and `Grafana <https://grafana.org/>`__. These integrations support data collection from several Mattermost servers, which is particularly useful if you're running Mattermost `in high availability mode </scale/high-availability-cluster.html>`__. Once you're tracking system health, you can `set up performance alerts </scale/performance-alerting.html>`__ on your Grafana dashboard.
+Performance monitoring support enables admins to track system health for large Enterprise deployments through integrations with `Prometheus <https://prometheus.io/>`__ and `Grafana <https://grafana.org/>`__. These integrations support data collection from several Mattermost servers, which is particularly useful if you're running Mattermost `in high availability mode </scale/high-availability-cluster.html>`__. Once you're tracking system health, you can `set up performance alerts </scale/performance-alerting.html>`__ on your Grafana dashboard.
 
-.. note::
+Admins can collect and store various data points from the Mattermost application in an `OpenMetrics <https://openmetrics.io>`__ format by `deploying the Mattermost Metrics plugin <#mattermost-metrics-plugin>`__, or by `deploying Prometheus <#install-prometheus>`__ and `Grafana <#install-grafana>`__.
 
-   While Prometheus and Grafana may be installed on the same server as Mattermost, we recommend installing these integrations on separate servers, and configure Prometheus to pull all metrics from Mattermost and other connected servers.
+Mattermost Metrics plugin
+-------------------------
+
+The Mattermost Metrics plugin, available from Mattermost v6.3, collects and stores the `same performance monitoring metrics <#statistics>`__ as if Prometheus and Grafana were deployed, without having to deploy these third-party tools. Data is collected every minute and is stored where the plugin is running. The data is sychronized to either a cloud-based or local file store every hour, and retained for 15 days. 
+
+Using the Mattermost Metrics plugin, you can download and share the collected data with Mattermost to understand application performance, troubleshoot system stability and performance, as well as inform route cause analysis.
+
+In High Availability deployments, the Mattermost Metrics Plugin operates in 2 modes: scraper mode and listener mode.
```

Comment:
@isacikgoz - Can you provide details on how admins can operate the plugin in each mode, please?

Reply:
@cwarnermm that's an implementation detail actually, admins have no control on these modes so maybe we can emphasize that or remove this information entirely. Both works for me.

Reply:
If admins don't have control over the modes, best to remove from docs.